stable/cerebro/Chart.yaml

```diff
@@ -6,7 +6,7 @@ sources:
  - https://github.com/yannart/docker-cerebro
  - https://github.com/lmenezes/cerebro
 name: cerebro
-version: 0.1.1
+version: 0.3.0
```

Comment:
Where did 0.2 go to?

Reply:
I tried to explain in the PR comment. See e.g. https://github.com/kubernetes/charts/pull/6595/commits/0dd516d41b9a4162e2581bcf8fbd930ee5aa32ed#diff-0fcab9ba30f2d528e9326bf1299ef520R9

Reply:
Basically wanted to make sure every commit (~ version) introduced by this PR has a correct Chart version attached to it.